client/strings/de.json

```diff
@@ -186,7 +186,7 @@
   "LabelAuthors": "Autoren",
   "LabelAutoDownloadEpisodes": "Episoden automatisch herunterladen",
   "LabelBackToUser": "Zurück zum Benutzer",
-  "LabelBackupLocation": "Backup Location",
+  "LabelBackupLocation": "Backup Ort",
```

Comment:
oder `Backup-Ort`?

vgl. [Leerzeichen in Komposita](https://de.wikipedia.org/wiki/Leerzeichen_in_Komposita) und  [Durchkopplung bei fremdsprachlichen Begriffen](https://de.wikipedia.org/wiki/Durchkopplung#Fremdsprachliche_Begriffe)

Reply:
ja, die Idee mit dem Bindestrich ist gut.